src/FixerConfiguration/AllowedValueSubset.php

```diff
@@ -24,6 +24,21 @@ public function __construct(array $values)
         $this->values = $values;
     }
 
+    public function __invoke($values)
```

Comment:
please document the method - purpose and interface

Reply:
Done.